src/Test/WebTestCase.php

```diff
@@ -31,7 +31,10 @@ protected static function createClient(array $options = [], array $server = [])
 
         if (static::$firstContainer === null) {
             static::$firstContainer = $client->getContainer();
-            static::initData($kernel);
+
+            if ((bool) $client->getContainer()->getParameter('testing.data_init')) {
```

Comment:
Parametr jsme udělali jako povinný? Šlo by ho udělat nepovinný s výchozí hodnotou?

Reply:
no pridal sem ho configu ve skeletonu, takze pri vytvoreni testovaciho projektu ho mas defaultne nastaveny a nemusis ho resit.

https://github.com/imatic/testing/blob/dbdde05d23f6f07741008bb54b69b462452889c2/resources/skeleton/TestsTemplate/Fixtures/TestProject/config/config.yml#L4

Reply:
Jasný, tak to asi stačí, díky!